### PR TITLE
VIRTS-550 Make sure "unsupported browser" notification stays while scrolling

### PR DESCRIPTION
**Pull request by Huzar01**
Added a few lines of code to **shared.css** and **shared.js** to make the "unsupported browser" notification banner stick on the page while scrolling.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,4 @@ data/payloads/*
 !data/payloads/.gitkeep
 data/facts/*yml
 !data/facts/.gitkeep
+conf/default.yml
```
Comment (privateducky) [inline .gitignore]: I don't think we want this line ^^ we want the default.yml to stay in the repo. other .yml files in conf, probably not though...

File: conf/default.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-port: 8888
+port: 9888
```
Comment (privateducky) [inline conf/default.yml]: don't want to change the ports here

**Comment by privateducky** [comment]
great idea-- just a few small mods that you prob didn't mean to add

File: .gitignore (hunk 1 above)
Comment (Huzar01) [reply .gitignore]: Yes I know.... I was trying to find a way to ignore that commit =/

File: conf/default.yml (hunk 2 above)
Comment (Huzar01) [reply conf/default.yml]: Same as above. Sorry about that!

**Pull request closed without merging**